.circleci/config.yml

```diff
@@ -455,6 +498,27 @@ workflows:
               ignore: /.*/
             tags:
               only: /^v.*/
+      - Focus build XCFramework:
+          # filters:
+          #   branches:
+          #     ignore: /.*/
+          #   tags:
+          #     only: /^v.*/ # We only build Focus on release since Focus
+              # only uses a subset of the components the full iOS build
+              # covers.
+          # TODO: Remove this before landing, temporary to test that CI
```

Comment:
Nit: Is this comment still accurate? bump here just to make sure we don't forget